### PR TITLE
LPS-175736 Created logic Apply Filter Button Action

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts [hunk 1]
```diff
@@ -70,14 +72,14 @@ const dataToOptions = <T = any>(
...
-		name: 'assignee',
+		name: 'assignedUsers',
 		resource: '/user-accounts',
 		transformData(item) {
 			return dataToOptions(
 				transformData<UserAccount>(item),
 				(userAccount) => ({
 					label: `${userAccount.givenName} ${userAccount.additionalName}`,
-					value: userAccount.id,
+					value: userAccount.givenName as string,
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts:82]:
```suggestion
					value: userAccount.givenName,
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Review by samuelleolopes — APPROVED** [review]

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:forward

**Review by welitonpe — APPROVED** [review]
LGTM :fire:

**Pull request closed without merging**